packages/start-aws/entry.ts

```diff
@@ -0,0 +1,64 @@
+import type { APIGatewayProxyEvent, APIGatewayProxyEventV2, APIGatewayProxyResult } from 'aws-lambda';
+import { splitCookiesString } from 'solid-start/node/fetch.js';
+import 'solid-start/node/globals.js';
+import { type FetchEvent } from 'solid-start';
+// @ts-ignore
+import { default as manifest } from '../../dist/client/route-manifest.json';
+// @ts-ignore
+import { default as _server } from './entry-server.js';
+const server = _server as (event:FetchEvent)=>Promise<Response>;
+
+export async function handler(
+  event:APIGatewayProxyEvent|APIGatewayProxyEventV2
+):Promise<APIGatewayProxyResult> {
+  const clientAddress =
+    (event as APIGatewayProxyEvent).requestContext.identity
```

Comment:
Suggestion: We can use typescript `in` operator instead of as here for better ts safety.